### DISCUSSION TITLE
Account Recovery

### DISCUSSION THREAD

**Discussion by DANS55** [mysteriumnetwork/node#4490] [General]
Hello MySteriumVpn customers,

I recharged my account today and my account became paid. The important thing is, I was trying to connect to the servers, but the connection was not accepted. I tried many ways, but it did not connect to any server.

I tried to clear the app's storage data, but I forgot to download the keystore

I am calling you now to retrieve it, please reply and help.

my identity address: 0x5f3504dd88bc649fad58f9a0bcc0fa7c990a51ca

**Comment by adinetech** [comment]
Sorry, it is not possible for us to recover, your Keystore (identity backup) has the lost MYST and we do not have access to it.
